src/ArticleCard.jsx

```diff
@@ -3,34 +3,52 @@ import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
+import { useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { CardActionArea } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import Article from './Article';
 
 
 
 const ArticleCard = (props) => {
     const { article } = props
+    const [id, setId] = useState(0)
+    const navigate = useNavigate();
+
+
+
+
     return (
         <Card sx={{ maxWidth: 225 }} className='item-card'>
-            <CardMedia
-                component="img"
-                height="100"
-                alt={article.title}
-                image={article.article_img_url}
-            />
-            <CardContent>
-                <Typography variant="h5" component="div" id="article_title">
-                    <li>{article.title}</li>
-                </Typography>
-                <Typography variant="h6" component="div" id="author">
-                    <li>{article.author}</li>
-                </Typography>
-                <Typography variant="h6" component="div" id="card-links">
-                    <li>{article.comment_count} comments</li>
-                    <li>{article.votes} votes</li>
-                </Typography>
-                <button id="read-more" onClick={() => { console.log(article.article_id) }}>Read more</button>
-            </CardContent>
+            <CardActionArea onClick={() => {
+                setId(article.article_id);
+                navigate(`/articles/${article.article_id}`);
```

Comment:
This should be a link not a navigate. Use navigate is for when we want our users to be directed programatically, such as a form, a login page or to a 404 if there is an error. If someone is being directed to click on something to move through a web application this should be a link. This is also more semantically readable to screen readers.